modules/ROOT/pages/custom-ingress-configuration.adoc

```diff
@@ -193,8 +193,21 @@ labels:
   environment: 
   environment-id
 ----
+|`unique-id` | `host` a| 
+
+[source,yaml]
+----
+host: app-name-unique-id.com
+----
 |===
 
+Use the `unique-id` placeholder to create a unique application URL, for example, if you want to deploy two applications with same name from different business groups.
```

Comment:
@kmendiratta-crm Do you foresee questions like:
1. How is this uniqueId created?
2. What kind of key it is ? (i mean its a shard string) etc

Reply:
@sarjeet2013

Reply:
I have updated the Notes section with example rendered value